hedy.py

```diff
@@ -823,6 +824,12 @@ def get_type(self, tree):
             if in_lookup:
                 return type_in_lookup
             else:
+
+                # if the variable name is 'at', it's because they tried to print "at random" 
+                # in a level that wasn't allowed
+                if var_name == 'at' and self.level >= 16:
```

Comment:
Do we need this? We already have the error rule right (and it will only work in En if we hard code 'at' like this)

Reply:
I was having some trouble with adding a rule for print in the lark file so did it this way instead. I looked at how "print fruit at random" was handled in previous levels, but it appears that when "list_access" was updated to now be "fruit[x]" opposed to "fruit at x," I couldn't correctly get the rule to work anymore. I agree that "At" will only work in English, so need to think of a different solution. 

I tried something like this 

```error_random_print: _PRINT (_print_argument_error)? -> error_random_keyword
_print_argument_error:  (_SPACE | quoted_text | list_access | expression | print_expression | at_random)``` but it didn't parse correctly. I feel like im a pretty close though?

Reply:
Maybe you could make a draft PR with the grammar attempt so I can take a look more easily?

Reply:
hi! sorry school ended and I went on vacation and then it was Christmas. I will take a look again soon- but for now converted it to draft!